From decdbc41a07d87321834be1cc10e18b11b84339c Mon Sep 17 00:00:00 2001
From: Ben Sartor <ben@simlar.org>
Date: Sat, 28 Mar 2020 17:35:45 +0100
Subject: [PATCH] [gradle] use bin as wrapper distribution for version 6.7.1

 * by running twice:
   ./gradlew wrapper --distribution-type bin --gradle-version=6.7.1 --gradle-distribution-sha256-sum=3239b5ed86c3838a37d983ac100573f64c1f3fd8e1eb6c89fa5f9529b5ec091d

 * for checksums see:
   https://gradle.org/release-checksums/
---
 gradle/wrapper/gradle-wrapper.properties | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gradle/wrapper/gradle-wrapper.properties b/gradle/wrapper/gradle-wrapper.properties
index cdc95ef6eb..8174453a1a 100644
--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=22449f5231796abd892c98b2a07c9ceebe4688d192cd2d6763f8e3bf8acbedeb
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
+distributionSha256Sum=3239b5ed86c3838a37d983ac100573f64c1f3fd8e1eb6c89fa5f9529b5ec091d
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-- 
2.20.1

